docs/docs/sso.mdx

```diff
@@ -212,7 +212,7 @@ In Azure, register an Application, instead of Enterprise, as we use OpenID Conne
   <div>
     <ol>
       <li>
-        Register an Application (we use OpenID Connect, so choose regular app, not Enterprise)
+        Register an Application (we use OpenID Connect, so choose regular app, not Enterprise). Because GrowthBook uses OIDC, ensure ID tokens are enabled, as the Microsoft Graph permission `User.read` does not work with OIDC.
```

Comment:
```suggestion
        Register an Application (we use OpenID Connect, so choose regular app, not Enterprise). Ensure ID tokens are enabled, as the Microsoft Graph permission `User.read` does not work with OIDC.
```

Reply:
I feel like that first phrase is not quite needed as in the parenthesis right before it we say "we use OpenID Connect", so no need to repeat it.  

I also don't quite understand the second bit, but maybe that's just cause I'm not staring at their UI.  I don't understand how having "ID tokens enabled" is related to Microsoft Graph permission `User.read`?  Does checking that disable Microsoft Graph?